acquire/uploaders/plugin.py

```diff
@@ -57,4 +63,7 @@ def upload_files_using_uploader(
                 log.error(*error_log)
                 log.exception("")
 
+        counter += 1
+        upload_gui.progress = 55 + (counter / len(paths) * 40)
```

Comment:
```suggestion
        upload_gui.progress = 55 + (counter // len(paths) * 40)
```
The type hint of `progress` is `int` but you set a float here.